### PR TITLE
new icon: reactrouter (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by Gautam25Raj**
* Add reactrouter icons

* Update devicon.json

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -8381,6 +8381,29 @@
...
+        "versions": {
+            "SVG": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ], 
+            "font": [
+                "original",
+                "original-wordmark"
```
Comment (Panquesito7) [inline devicon.json:8401]: The font versions should always (or in most cases) be the plain versions.
```suggestion
                "plain",
                "plain-wordmark"
```

File: devicon.json [hunk 2]
```diff
@@ -8381,6 +8381,29 @@
             }
         ]
     },
+    {
+        "name": "reactrouter", 
+        "altnames": [
+            "reactrouter"
+        ],
```
Comment (Panquesito7) [inline devicon.json:8388]: If the alternative name's the same as the name, it shouldn't be added.
```suggestion
        "altnames": [],
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: devicon.json [hunk 3]
```diff
@@ -8381,6 +8381,28 @@
...
     },
+    {
+        "name": "reactrouter", 
+        "altnames": [],
+        "tags": [
+            "framework"
+            "react"
+        ], 
+        "versions": {
+            "SVG": [
```
Comment (Panquesito7) [inline devicon.json:8392]:
```suggestion
            "svg": [
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🚀

image

image

**Comment by lunatic-fox** [comment]
Just to be extra sure after the modification.
Color `#f44250` will be added by `color` property of `devicon.json`.

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**